frontend/src/Pages/Home.tsx

```diff
@@ -2,21 +2,36 @@ import "./Home.css";
 // import { useNavigate } from "react-router-dom";
 import FeedContainer from "../PageElements/Home/Feed/feed-container";
 import InputContainer from "../PageElements/Home/Feed/input-container";
+import PeopleBox from "../PageElements/Home/Sidebar/people-sidebar";
 
 const Home = () => {
     return (
-        <div className="HomePage">
-            <div className="HomePage-header"></div>
-            <div className="HomePage-columnLeft">
-                <li/>
-                <InputContainer/>
-                <li/>
-                <FeedContainer/>
+        <div className="min-h-screen flex flex-col">
+            {/*Main Page Container*/}
+            <div className="bg-gray-500 py-5">
+                {/*Header Section*/}
             </div>
-            <div className="HomePage-columnRight">
-                
+
+            <div className="flex flex-grow">
+                {/*Main Body Sections*/}
+
+                <div className="bg-gray-900 w-3/4 flex flex-grow">
+                    {/*Left Column Section*/}
+                    <ul className="w-full p-5 items-center text-center">
+                        <InputContainer/>
+                        <FeedContainer/>
+                    </ul>
+                </div>
+
+                <div className="w-1/4 bg-gray-700 p-4">
+                    {/*Right Column Section*/}
+                    <PeopleBox/>
```

Comment:
I think it might be better to use the sidebar container here rather than just one individual box. If you don't like the colors I can change them to fit better

Reply:
I agree. This was just placed for visualization of the PeopleBox to make sure it worked. I'll modify it!